packages/i/imagemagick/xmake.lua

```diff
@@ -70,6 +71,11 @@ package("imagemagick")
         if package:config("pic") ~= false then
             table.insert(configs, "--with-pic")
         end
+        if package:config("threads") then
+            package:add("cflags", "-pthread")
```

Comment:
add_syslinks

Reply:
`-pthread` will already add `-lpthread`, and apparently just adding `-lpthread` rarely helps as it only links the lib and not any macros or anything.

I'll change it to add_syslinks though and place it out of the on_install function.